src/shared/locales/en/translation.json

```
@@ -127,6 +127,7 @@
         "nodeDuplicatedExplanation": "The custom node is already listed.",
         "nodeAutoChanged": "Auto-switched node",
         "nodeAutoChangedExplanation": "Switched to new node {{- nodeAddress}} as selected node is offline.",
+        "touchSensor": "Touch the fingerprint sensor",
```

```suggestion
        "touchSensor": "Touch the fingerprint reader",
```
Just being nitpicky :trollface: we use `reader` in other places

---

@rajivshah3 It says sensor here: https://material.io/design/platform-guidance/android-fingerprint.html#authentication

---

We use reader here:
https://github.com/iotaledger/trinity-wallet/blob/4bba5d4931e6aac2ab578fc2ae21b16c77b2cf4b/src/shared/locales/en/translation.json#L131